models/sibcasa/DESCRIPTION

```diff
@@ -2,7 +2,6 @@ Package: PEcAn.SIBCASA
 Type: Package
 Title: PEcAn Package for Integration of the SiBCASA Model
 Version: 0.0.1
```

Comment:
This is the first release of PEcAn.SIBCASA, but it's not stable and not well-tested. Leaving the version at 0.0.1 to reflect that.